llvm/test/tools/llvm-objcopy/ELF/add-invalid-note.test

```diff
@@ -0,0 +1,35 @@
+# Verify that --add-section can be used to add a note section which is
```

Comment:
Nit: use `##` for comments in new test files.

This comment doesn't make sense though: this test is about adding invalid note sections...